src/NuGet.Core/NuGet.Common/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,5 @@
-NuGet.Common.NuGetLogCode.NU5049 = 5049 -> NuGet.Common.NuGetLogCode
\ No newline at end of file
+NuGet.Common.FileSystemInfoFullNameEqualityComparer
```

Comment:
Feels like the ordering is off here :D 
I imagine it's supposed to be alphabetical.

Reply:
I found a flaw in the logic around when a file is created for the first time vs when its just opened so I'm going to revert this public API change in favor of a `lock` instead of a `SemaphoreSlim`.

Reply:
Resolving this.